RELEASE.md

```diff
@@ -397,12 +397,14 @@ or create your own locally with:
 cd dangerzone
 ```
 
-Build the latest container:
+Build the latest container, on both architectures:
```

Comment:
Something tells me that this should go to the macOS section, right?

Reply:
Yup!